### PR TITLE
Release smart_proxy_dynflow 0.0.6 (DEB)

### PR DESCRIPTION
**Pull request by stbenjam**
Nightly + 1.10

### CODE REVIEW

File: plugins/smart_proxy_dynflow/postinst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+set -e
+
+USERNAME="foreman-proxy"
+GROUPNAME="foreman-proxy"
+DYNFLOW_DB_DIR="/var/lib/foreman-proxy/dynflow"
+
+case "$1" in
+  configure)
+    mkdir -p $DYNFLOW_DB_DIR
+    chown $USERNAME:$GROUPNAME $DYNFLOW_DB_DIR
```
Comment (domcleal) [inline plugins/smart_proxy_dynflow/postinst]: Could you also set a restrictive mode for the DB here?  It'll prevent any user reading the DB.

Comment (stbenjam) [inline plugins/smart_proxy_dynflow/postinst]: Done below, 750 ok?

Comment (domcleal) [inline plugins/smart_proxy_dynflow/postinst]: Yes, I think 750 will be fine.

**Comment by mmoll** [comment]
cherry-picked as 64ebcc0 for develop and as 07adea7 for 1.10, thanks @stbenjam!

**Pull request closed without merging**